packages/core-utils/src/__tests__/route.js

```diff
@@ -135,4 +136,60 @@ describe("util > route", () => {
       expect(sortRoutes(route12, route13)).toMatchSnapshot();
     });
   });
+
+  describe("route text color generation", () => {
+    it("should not modify appropriate color pairings", () => {
+      expect(getMostReadableTextColor("#EE352E", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#00933C", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#b933ad", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#808183", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#0039a6", "#ffffff")).toBe("#ffffff");
+
+      // Both work
+      expect(getMostReadableTextColor("#ff6319", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#ff6319", "#000000")).toBe("#000000");
+      expect(getMostReadableTextColor("#6cbe45", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#6cbe45", "#000000")).toBe("#000000");
+      expect(getMostReadableTextColor("#a7a9ac", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#a7a9ac", "#000000")).toBe("#000000");
+      expect(getMostReadableTextColor("#FF00FF", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#FF00FF", "#000000")).toBe("#000000");
+
+      expect(getMostReadableTextColor("#996633", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#fccc0a", "#000000")).toBe("#000000");
+
+      expect(getMostReadableTextColor("#0075B2", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#D4A723", "#000000")).toBe("#000000");
+      expect(getMostReadableTextColor("#009D4B", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#CE242B", "#ffffff")).toBe("#ffffff");
+
```

Comment:
Add some tests with a proposed color that is not black or white.

Reply:
Couldn't find any agencies with non-black/white pairings. I have one at the end that has a non-bw color. I've also added an interactive way to test the method which allows you to test any color.